src/components/learner-credit-management/MemberStatusTableColumnHeader.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {
+  IconButtonWithTooltip,
+  Stack,
+  Icon,
+} from '@edx/paragon';
+import { InfoOutline } from '@edx/paragon/icons';
+
+const MemberStatusTableColumnHeader = () => (
+  <Stack gap={0} direction="horizontal">
+    <p data-testid="members-table-status-column-header" className="mb-0 mr-1">Status</p>
+    <IconButtonWithTooltip
```

Comment:
Based on figma it doesn't look like we're using this component because the icon looks smaller and doesn't have a seperate active state. I would mimic what I did on the budget headers [here](https://github.com/openedx/frontend-app-admin-portal/blob/8e145f93780545ed352fe6f091e677b84c46fe6d/src/components/learner-credit-management/BudgetStatusSubtitle.jsx#L32)